cmd/loop/quote.go

```diff
@@ -12,12 +12,25 @@ var quoteCommand = cli.Command{
 	Usage:       "get a quote for the cost of a swap",
 	ArgsUsage:   "amt",
 	Description: "Allows to determine the cost of a swap up front",
-	Action:      quote,
+	Flags: []cli.Flag{
+		cli.Uint64Flag{
+			Name: "conf-target",
+			Usage: "the confirmation target that should be used " +
+				"either for the sweep of the on-chain HTLC " +
+				"broadcast by the swap server in the case of " +
+				"a Loop Out, or for the confirmation of the " +
+				"on-chain HTLC broadcast by the swap client " +
+				"in the case of a Loop In",
```

Comment:
Description may be confusing because there is no way yet to control loop in chain fees.

Reply:
Removed loop in portion.